### PR TITLE
Assignment 1 - nparkman

### PR DESCRIPTION
**Pull request by kinpark**
Didn't realize we were supposed to do this last weekend. Been also experiencing some technical issues getting everything to work.  Finally got my stuff to deploy to my site... need to fix code now i guess. Hope I'm doing this pull request right...

### CODE REVIEW

**Comment by BenKnoll3** [comment]
Create a Vue app ✅

Uses TypeScript ✅
Add Vuetify ✅
Uses Vitest ✅
Create a simple page called game.vue ✅

Page should have a card with a heading that with a clever name for the game ❌
some default placeholder body text ❌
and a button to take you back to the home page ❌
there should be a button on the home page to access this page ✅
Check all the code into your repo fork ✅

Create a static web app in Azure ✅

Create the container where your GitHub repo will deploy
Link the site with your GitHub repo
Update the CI/CD pipeline ✅

Pipeline runs on each check in
Vitest run and pass ✅
Vue app builds ✅
Vue app deploys to Azure ✅
Azure site works ✅

Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅
Extra Credit
Come up with up to three unique ideas for stories we could implement as part of our project. Post these in your readme.md. ✅
One point per idea up to three points

- I believe we are supposed to pull request our Assignment-1 branch against their Assignment-1 branch so that the git diff is easier to read.
- I see that you are further along in class with your code but you don't have the v-card with default placeholder text, and a button to take you back to the home page.

Everything else looks good, and I like the extra credit ideas!

**Review by BenKnoll3 — COMMENTED** [review]
Create a Vue app ✅

Uses TypeScript ✅
Add Vuetify ✅
Uses Vitest ✅
Create a simple page called game.vue ✅

Page should have a card with a heading that with a clever name for the game ❌
some default placeholder body text ❌
and a button to take you back to the home page ❌
there should be a button on the home page to access this page ✅
Check all the code into your repo fork ✅

Create a static web app in Azure ✅

Create the container where your GitHub repo will deploy
Link the site with your GitHub repo
Update the CI/CD pipeline ✅

Pipeline runs on each check in
Vitest run and pass ✅
Vue app builds ✅
Vue app deploys to Azure ✅
Azure site works ✅

Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅
Extra Credit
Come up with up to three unique ideas for stories we could implement as part of our project. Post these in your readme.md. ✅
One point per idea up to three points

-I believe we are supposed to pull request our Assignment-1 branch against their Assignment-1 branch so that the git diff is easier to read.
-I see that you are further along in class with your code but you don't have the v-card with default placeholder text, and a button to take you back to the home page.

Everything else looks good, and I like the extra credit ideas!

File: wordle-web/README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+#Wordle Site Link
+https://brave-forest-08291851e.3.azurestaticapps.net 
+
+#EC Story Ideas
```
Comment (mmwoodfo) [inline wordle-web/README.md:4]: Extra credit was specifically for ideas related to the wordle game, but that's okay!

**Review by mmwoodfo — COMMENTED** [review]
I don't see your Game.vue page?

**Comment by mmwoodfo** [comment]
Your PR should be against the Assignment 1 branch

**Pull request closed without merging**